.github/workflows/ci.yml

```diff
@@ -19,8 +19,8 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
-    - run: npm i -g yarn@latest
-    - run: npm i -g yarn@1
+    - run: yarn set version stable
```

Comment:
its odd to call yarn before it was installed. This assumes that Yarn was already available before? can we flip?